docs/tutorials.superdense_coding.rst

```diff
@@ -1,4 +1,150 @@
 Superdense Coding
 ==================
 
-Coming soon.
\ No newline at end of file
+In classical communication, sending two bits of information requires transmitting
+two physical bits. But with the help of quantum mechanics we can bend this rule
+using the phenomenon of quantum entanglement. **Superdense coding** proposed by
+Bennet and Wiesner in 1992 :cite:`Bennett_1992_Communication` lets Alice send two classical bits to
+Bob by transmitting just *one qubit* . The catch here is that they must share an
+entangled pair of qubits beforehand. We will explain this protocol in detail 
+below:
+
+Superdense Coding protocol
```

Comment:
```suggestion
Superdense coding protocol
```

(to maintain consistency with the capitalization of things as you write them above).